구현_실습/Boilerplate/src/css/style.css

```
@@ -0,0 +1,38 @@
+.hide {
+    display: none;
+}
+
+ul.todo-list {
+    padding-left: 0px;
+}
+
+li.todo-item {
+    list-style: none;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+input:checked + label {
+    color: #808080;
+    text-decoration: line-through;
+}
+
+div.status > div {
+    display: inline-block;
+    cursor: pointer;
+    margin: 5px;
+}
+
+div.status__filterBtn__checked {
+    border: 1px solid;
+    border-radius: 5px;
+    border-color: #808080;
```

```suggestion
    border: 1px solid #808080;
    border-radius: 5px;
```
단축 사용하는 김에 `color`도 합치면 좋을 것 같습니다.